bin/rucio

```diff
@@ -358,16 +359,9 @@ def list_file_replicas(args):
 
     table = []
     dids = []
-    rse_dict = {}
     if args.missing and not args.selected_rse:
         print 'Cannot use --missing without specifying a RSE'
         return FAILURE
-    if args.selected_rse and args.list_collections:
```

Comment:
are these removals on purpose?

Reply:
Yes, the option was deprecated for a long time :
    list_file_replicas_parser.add_argument('--list-collections', dest='list_collections', action='store_true', default=False, help='To have the number of files of the dataset by site\
        (' + Color.BOLD + 'DEPRECATED: ' + Color.END + 'please use list-dataset-replicas instead.)', required=False)